tfe-scripts/tf-update-variable-values.sh

```diff
@@ -20,14 +20,61 @@ else
     jq -r '.data.id'))
 fi
 
+update_value () {
+  ./tfe-scripts/tf-update-variable-value.sh "$WORKSPACE_ID" "$1" "$2"
+}
+
 while IFS= read -r line; do
-  VAR_KEY="$(cut -d'=' -f1 <<<"$line")"
-  VAR_VAL="$(cut -d'=' -f2- <<<"$line")"
+  var_key="$(cut -d'=' -f1 <<<"$line")"
+  var_val="$(cut -d'=' -f2- <<<"$line")"
+
+  if [ ! -z "$var_key" ] && [ ! -z "$var_val" ]; then
+    if [ $var_key == "namespace_apps" ]; then
+      # create double quote escaped namespace_apps array string and collect namespaces
+      namespaces=()
+      namespace_apps="["
+      i=0
+      for value in $(echo "${var_val}" | jq -r '.[]'); do
+        namespaces+=( "$(cut -d',' -f1 <<<"$value")" )
+        if [ $i -ne 0 ]; then namespace_apps=$namespace_apps","; fi
```

Comment:
```suggestion
        if [ $i -ne 0 ]; then namespace_apps="${namespace_apps},"; fi
```
possible quoting improvement (why I'm suggesting linter)

Reply:
I will add Bats tests here as well!